# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/nsCSSValue.h
# Commit: a16d763f2866
# Full Hash: a16d763f2866a1c44e67b85a66fa4503b5c764ac
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-09-13 09:25:56
# Regressor Bug: 1287054
# File Overlap Count: 1
# Description:
#   Bug 1287054 part 1 - support rem, ch, ic and cap units for non-CSS lengths r=emilio,jgilbert
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D180618
# ==============================================================================

diff -r ccfee1091860 -r a16d763f2866 layout/style/nsCSSValue.h
--- a/layout/style/nsCSSValue.h	Tue Sep 12 20:57:49 2023 +0000
+++ b/layout/style/nsCSSValue.h	Tue Sep 12 21:10:54 2023 +0000
@@ -61,6 +61,8 @@
   eCSSUnit_Char = 802,     // number of characters, used for width with
                            // monospace font
   eCSSUnit_RootEM = 803,   // == root element font size
+  eCSSUnit_Ideographic = 804,  // == CJK water ideograph width
+  eCSSUnit_CapHeight = 805,    // == Capital letter height
 
   // Screen relative measure
   eCSSUnit_Point = 900,       // 4/3 of a CSS pixel